modules/cli/src/main/resources/META-INF/native-image/org.virtuslab/scala-cli-core/resource-config.json

```diff
@@ -21,6 +21,9 @@
       },
       {
         "pattern": "\\Qcom/google/javascript/jscomp/js/polyfills.txt\\E"
+      },
+      {
+        "pattern": ".*scala3RuntimeFixes.jar$"
```

Comment:
It seems this is redundant with the `-H:IncludeResources=` added in `project/settings.sc`. Better keep things there, as this should allow to keep the resource name in a single place.